host/sgx/sgxquote.c

```diff
@@ -135,6 +136,191 @@ static quote3_error_t (*_sgx_qv_verify_quote)(
 static void* _ql_module;
 static void* _qvl_module;
 
+// This is a helper for getting "human readable error messages" - as mentioned
+// in issue 2945 - so that we do not have to return complex quote3_error_t codes
```

Comment:
I don't think we should cite issue #'s except in TODO's.  Since 2945 is closed, it would be more appropriate to copy any relevant information into the comments here.

Reply:
Updated.